# HG changeset patch
# User David Meyer <pdox@google.com>
# Date 1306440127 25200
# Branch pnacl-sfi
# Node ID 7cf9f3a52aaf5630750b55ba713ac8aa3b685e35
# Parent 4332a4ccc6be8171bc2bbdd62d69afcbc8647bf9
Disable checking for symbol __nacl_read_tp (issue7071010)

 From llvm-pnacl-0036-260-7cf9f3a52aaf5630750b55ba713ac8aa3b685e35.patch

 From .hg/patches/llvm-pnacl-0036-260-7cf9f3a52aaf5630750b55ba713ac8aa3b685e35.patch.stripped

diff -r 4332a4ccc6be lib/CodeGen/IntrinsicLowering.cpp
--- a/lib/CodeGen/IntrinsicLowering.cpp	Fri May 20 16:06:57 2011 -0700
+++ b/lib/CodeGen/IntrinsicLowering.cpp	Thu Jun 30 00:22:43 2011 -0700
@@ -129,8 +129,11 @@
   // assert that __nacl_read_tp exists.
   // Insert a correctly-typed definition now.
   {
-    const Type *RetTy = Type::getInt8PtrTy(M.getContext());
-    M.getOrInsertFunction("__nacl_read_tp", RetTy, (Type*)0);
+    // @TODO(pdox): Re-enable checking for __nacl_read_tp when we
+    //              build glibc ourselves.
+
+    //const Type *RetTy = Type::getInt8PtrTy(M.getContext());
+    //M.getOrInsertFunction("__nacl_read_tp", RetTy, (Type*)0);
   }
   // @LOCALMOD-END
 
